### ISSUE TITLE
Create a faucet website for node testing

### ISSUE THREAD

**Issue by phklive** [0xPolygonMiden/miden-node#215]
For testing purposes it would be useful to be able to initialise that node and make a simple frontend available where any user could input their account ID and get a note generated with some amount of tokens. 

When initialising the node for testing the faucet could be automatically created and the faucet website started. We could have both the node and website living on the same machine. 

**Goal**: Create a simple website frontend faucet enabling users to get assets.

**Comment by phklive** [comment]
Depends on: #223

**Comment by Dominik1999** [comment]
Will this happen this week?

**Comment by phklive** [comment]
> Will this happen this week?

That's the goal, working on it actively.

**Comment by Dominik1999** [comment]
This is done, right?

https://ethdenver.polygonmiden.io/

**Comment by phklive** [comment]
It works indeed but the code has not yet been merged with the miden-node repo. Will close once it's merged.

https://github.com/0xPolygonMiden/miden-node/pull/247

**Comment by gubloon** [comment]
> This is done, right?
> 
> https://ethdenver.polygonmiden.io/

@Dominik1999 
Can we repurpose this domain(polygonmiden.io) for playground or if we are not going to use http://miden.io/ anymore?

**Comment by Dominik1999** [comment]
we can use miden.io indeed.

**Comment by Dominik1999** [comment]
> It works indeed but the code has not yet been merged with the miden-node repo. Will close once it's merged.
> 
> [#247](https://github.com/0xPolygonMiden/miden-node/pull/247)

It looks like the PR was closed by not merged. Can you update this issue here accordingly?

**Comment by phklive** [comment]
Yes this PR has not yet been merged it needs some more cleanup, will merge by this week.

This is the new PR: #270

**Comment by Dominik1999** [comment]
What is missing here? Can we merge that today?

**Comment by phklive** [comment]
> What is missing here? Can we merge that today?

This PR is ready to be reviewed, waiting for an approval to merge.

**Comment by Dominik1999** [comment]
Closed by #270
